modules/jupyter/authentication/authenticator/gcpiapjwtauthenticator/gcpiapjwtauthenticator.py

```diff
@@ -59,33 +95,48 @@ class GCPIAPAuthenticator(Authenticator):
     header_name = Unicode(
         config=True,
         help="""HYYP header to inspect for the authenticated JWT.""")
```

Comment:
Could you do a quick fix here? It should be 'HTTP' instead of 'HYYP'

Reply:
LOL, just notice, for sure